bindings/python/convert.py

```diff
@@ -156,7 +163,7 @@ def previous_pr(api: "HfApi", model_id: str, pr_title: str) -> Optional["Discuss
             return discussion
 
 
-def convert(api: "HfApi", model_id: str, force: bool=False) -> Optional["CommitInfo"]:
+def convert(api: "HfApi", model_id: str, force: bool = False) -> Optional["CommitInfo"]:
```

Comment:
Didn't catch this before but we shouldn't use any api object and just the top level methods (`create_commit` is directly in the main init).

Reply:
Why ?

I figured it was actually better to pass around an HfApi codewise, so you can include a specific token (for instance to create conversions directly).
The original space convertion was passing around a Token, but I figure `HfApi` was even easier to transport around and there's less risk to forget to include `token=token` in whatever call.

Reply:
I don't think anyone who uses hugginface_hub uses the HfApi object.

Reply:
You should include a kwarg for token instead IMO

Reply:
> I don't think anyone who uses hugginface_hub uses the HfApi object.

I do  :)

Reply:
(this feature of setting a token on HfApi is quite recent though, but it's useful IMO)

Reply:
Let me rephrase, I don't think many people except Julien use `HfApi` :-p

Reply:
I did use it naturally actually an I think it's pretty cool to encapsulate common client options (most important of which the token).

I really think the "oh but you forgot to pass the token" is a real failure case, and it did happen when I used `get_repo_discussions` where I failed to pass the token and it would fail on private models on the space.

Reply:
I'll go ahead and merge this.

Please open an issue if you feel so strongly about the `HfApi` object :)